sys-libs/libatomic_ops/libatomic_ops-7.4.2.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided \
+             atomic memory update operations on a number architectures \
+             as said on the github project home."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops/"
+COPYRIGHT="1989, 1991 Free Software Foundation, Inc."
```

Comment:
You have put copyright of the license here, not of the code.
By inspecting the source files, I have found that the proper copyright should be `Hewlett-Packard Development Company, L.P.`. Try to find the year range.